kinto/docker-compose.yml

```diff
@@ -1,36 +1,8 @@
-db1:
-  image: postgres
-  environment:
-    POSTGRES_USER: postgres
-    POSTGRES_PASSWORD: postgres
-db2:
-  image: postgres
-  environment:
-    POSTGRES_USER: postgres
-    POSTGRES_PASSWORD: postgres
 master:
   image: kinto/kinto-server
-  links:
-   - db1
   ports:
    - "8888:8888"
-  environment:
-    KINTO_CACHE_BACKEND: cliquet.cache.postgresql
-    KINTO_CACHE_URL: postgres://postgres:postgres@db/postgres
-    KINTO_STORAGE_BACKEND: cliquet.storage.postgresql
-    KINTO_STORAGE_URL: postgres://postgres:postgres@db/postgres
-    KINTO_PERMISSION_BACKEND: cliquet.permission.postgresql
-    KINTO_PERMISSION_URL: postgres://postgres:postgres@db/postgres
 read-only:
```

Comment:
I ran this through some random Docker Compose YAML validator (https://lorry.io/) and it came back with a possible error on the `read-only` line:

> Invalid service name. Valid characters are [a-zA-Z0-9]

Me thinks it doesn't like the hyphen, but I haven't tried digging into some Compose spec to see if they specify character classes.